assets/scss/_custom.scss

```diff
@@ -530,6 +530,14 @@ body {
     border-left-color: #444;
   }
 
+  .alert-info:has(h4):has(ul) h4 {
+    line-height: 0 !important;
+  }
+ 
+  .alert-info:has(h4):has(ul) ul {
+    margin-top: 10px;
```

Comment:
Try this:
```suggestion
  float: initial;
  line-height: inherit;
  margin-bottom: 1rem;
```

Reply:
@sftim 
it looks good if only add [float: initial !important;]

![image](https://github.com/kubernetes/website/assets/20103020/76c0c2db-fc64-476f-96d8-6b84471f4c42)

Reply:
@sftim 
committed new changes please review it.